code/controllers/subsystem/rogue/devotion.dm

```diff
@@ -25,7 +25,7 @@
 	/// Current progression (experience)
 	var/progression = 0
 	/// Maximum progression (experience) we can achieve
-	var/max_progression = CLERIC_REQ_4
+	var/max_progression = CLERIC_REQ_2
```

Comment:
why not make this 3? The issue is revivals no?

Reply:
mind you stuff like  attach body part. Is like...borderline just ''its cool'' Anyone with lesser miracle can do it. Just click arm onto person and heal until its fully attached.

Reply:
The reason is essentially revivals, which I believe are available at level 3 with Astrata, which is why I went with 2. I could be wrong, I'll double-check?

Reply:
if thats the case switch it? Flip 3 and 4 for them? Seems silly to balance every single type based on a single.

Reply:
Yeah that seems nicer to me. Could even reduce the devotion unlock for T4 abit (so church does not see this as a nerf for them). Move Astrata Revival to T4. Then? Cure rot and revival are church only. Druids, eora and Pestra people don't lose out on their funni spells that are not centered around revival.  Everyone hopefully happy?

Reply:
Now looking through it again, a large part of it was to remove Conjure Glowshroom from them because it's a wretched spell for PvE content, which clerics engage in a lot in as adventurers.